Ring/event stability (#34644)
home-assistant/home-assistant.github.io: source/_integrations/ring.markdown

The event entity captures events like doorbell rings, motion alerts, and intercom unlocking.

### Realtime event stability

If you are experiencing issues with receiving ring alerts the reason could be that you have too many authenticated devices on your ring account.

> Prior to version 2023.12.0, the Home Assistant ring integration would register a new entry in `Authorized Client Devices` in the `Control Centre` at [ring.com](https://account.ring.com/account/control-center/authorized-devices) every time it restarted.

Reviewer: ```suggestion
If you are experiencing issues with receiving ring alerts, the reason could be that you have too many authenticated devices on your ring account.
Prior to version 2023.12.0, the Home Assistant ring integration would register a new entry in `Authorized Client Devices` in the `Control Centre` at [ring.com](https://account.ring.com/account/control-center/authorized-devices) every time it restarted.
```

> If you have been using the ring integration prior to this you may have many `Authorized Client Devices` in the `Control Centre` on [ring.com](https://account.ring.com/account/control-center/authorized-devices).

Reviewer: ```suggestion
If you have been using the ring integration before this, you may have many `Authorized Client Devices` in the `Control Centre` on [ring.com](https://account.ring.com/account/control-center/authorized-devices).
```
